internal/kafka/command_mirror_resume.go

```diff
@@ -108,33 +117,43 @@ func printAlterMirrorResult(cmd *cobra.Command, results []kafkarestv3.AlterMirro
 			}
 		}
 
-		for _, partitionLag := range result.GetMirrorLags().Items {
-			partitionId := partitionLag.GetPartition()
-			if isTruncateAndRestore && truncationData[partitionId] != nil {
-				list.Add(&mirrorOut{
-					MirrorTopicName:       result.GetMirrorTopicName(),
-					Partition:             partitionId,
-					ErrorMessage:          errorMessage,
-					ErrorCode:             errorCode,
-					PartitionMirrorLag:    partitionLag.GetLag(),
-					LastSourceFetchOffset: partitionLag.GetLastSourceFetchOffset(),
-					MessagesTruncated:     truncationData[partitionId].GetMessagesTruncated(),
-					OffsetTruncatedTo:     strconv.FormatInt(truncationData[partitionId].GetOffsetTruncatedTo(), 10),
-				})
-			} else {
-				list.Add(&mirrorOut{
-					MirrorTopicName:       result.GetMirrorTopicName(),
-					Partition:             partitionLag.GetPartition(),
-					ErrorMessage:          errorMessage,
-					ErrorCode:             errorCode,
-					PartitionMirrorLag:    partitionLag.GetLag(),
-					LastSourceFetchOffset: partitionLag.GetLastSourceFetchOffset(),
-				})
+		if !isTruncateAndRestore || includePartitionLevelTruncationData {
+			for _, partitionLag := range result.GetMirrorLags().Items {
```

Comment:
Is the terminology here "partition lag" or "mirror lag"?

Reply:
It's technically mirror lag, will update the terminology.

Reply:
It can be referred to either, though I think we normally use mirror lag.